src/target/linux.rs

```diff
@@ -169,6 +169,11 @@ mod tests {
     use super::ReadMemory;
     use nix::unistd::getpid;
 
+    extern crate alloc;
+    use alloc::alloc::{alloc_zeroed, Layout};
```

Comment:
I believe this can be replaced with
```suggestion
    use std::alloc::{alloc_zeroed, Layout};
```
and then we won't need `extern crate alloc`